NineChronicles.Headless/GraphTypes/Notification.cs

```diff
@@ -5,13 +5,9 @@ public class Notification
         public NotificationEnum Type { get; set; }
         public string Message { get; set; }
 
-        public Notification(NotificationEnum type)
-        {
-            Type = type;
-        }
-
-        public Notification(NotificationEnum type, string msg) : this(type)
+        public Notification(NotificationEnum type, string msg = "")
```

Comment:
Can we use `string.Empty`?

Reply:
No, we can't. Because `string.Empty` is not contant value, it can't be used as `default` value.